lib/diagnostic.rb

```diff
@@ -100,25 +105,30 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = 'arr.drop_while { |a| a > 34 }'
```

Comment:
Yes this would work. But to ALWAYS drop the last two from an array, you could do:
```ruby
arr.pop 2
```